src/aioice/turn.py

```diff
@@ -183,7 +183,8 @@ async def delete(self) -> None:
             pass
 
         logger.info("TURN allocation deleted %s", self.relayed_address)
-        self.transport.close()
+        print("cloose", self.transport)
```

Comment:
This looks like a typo: cloose -> close

Reply:
This is WIP, there will be no `print()` statements once I figure out what's going on!